UserGuide.md

```diff
@@ -154,23 +157,25 @@ The best way to create tests for CARROT is to start with a WDL that already exis
 
 In addition, another WDL should be written or adapted from existing tests which will run a comparison between the metrics outputs of the **_test WDL_** and precalculated baseline metrics.  We refer to this WDL as the **_evaluation WDL_**.
 
-Alternatively you can leave the structure of your original tool WDL as-is, using it as the test WDL ([with some modifications detailed below](#wdl-requirements)), and implement all metrics creations and comparison in your evaluation WDL
+Alternatively you can leave the structure of your original tool WDL as-is, using it as the test WDL, and implement all metrics creations and comparison in your evaluation WDL
 
 
 #### <a name="wdl-requirements"/> **WDL Requirements**
 
 
 ##### <a name="permissions-access"/> **Permissions / Access**
 
-If you supply any of the test resources (WDLs or their dependencies) via http/https/gs URI, the CARROT service account must have access to the WDLs and WDL resources to run for your test.  Because of this, <span style="text-decoration:underline;">you must make your WDLs and all associated required resources public or accessible by the service account that CARROT uses to retrieve and run WDLs</span>.
+If you supply any of the test resources (WDLs or their dependencies) via http/https/gs URI, the CARROT service account must have access to the WDLs and WDL resources to run for your test.  Because of this, you must make your WDLs and all associated required resources public or accessible by the service account that CARROT uses to retrieve and run WDLs.
+
+You also have the option of supplying your WDLs and their dependencies as files when creating or updating a template.
 
 
 #### <a name="input-requirements"/> **Input Requirements**
 
 
 ##### <a name="mapping-test-outputs-to-eval-inputs"/> **Mapping Test Outputs to Eval Inputs**
 
-It is encouraged to map outputs from your test WDL to inputs for your evaluation WDL.  To do so requires using a specific format for the value of the input supplied to your evaluation WDL.  For example, if your test WDL defines a workflow named “Test_Workflow”, which has an output called “output_vcf”, and you want to supply that as an input called “test_vcf” to a workflow called “Eval_Workflow” defined in your evaluation WDL, your input json for the evaluation WDL should contain the following:
+In order to evaluate the outputs of your test WDL in your evaluation WDL, you'll need to map those outputs to the evaluation WDL's inputs.  To do so requires using a specific format for the value of the input supplied to your evaluation WDL.  For example, if your test WDL defines a workflow named “Test_Workflow”, which has an output called “output_vcf”, and you want to supply that as an input called “test_vcf” to a workflow called “Eval_Workflow” defined in your evaluation WDL, your input json for the evaluation WDL should contain the following:
```

Comment:
Is there any sort of checker/linter that wdl writers can use to validate that these line up?

Reply:
There isn't, but that's a good idea.  I'll create an issue for that.